File: spring/mapstruct-spring-extensions-example/src/main/java/com/github/yukihane/spring/mapstructspringextensionsexample/WheelMapper.java
package com.github.yukihane.spring.mapstructspringextensionsexample;

import org.mapstruct.Mapper;
import org.mapstruct.Mapping;

@Mapper
public interface WheelMapper {
    @Mapping(source = "size", target = "wheelSize")
    WheelDto convert(Wheel wheel);
}

File: spring/mapstruct-spring-extensions-kotlin/src/main/kotlin/com/example/mapstruct_spring_extensions_kotlin/WheelMapper.kt
package com.example.mapstruct_spring_extensions_kotlin

import org.mapstruct.Mapper
import org.mapstruct.Mapping

@Mapper
interface WheelMapper {
    @Mapping(source = "size", target = "wheelSize")
    fun convert(wheel: Wheel): WheelDto
}
